src/content/docs/service-level-management/create-slm.mdx

```diff
@@ -423,7 +427,7 @@ Follow these steps:
 
 ### Edit SLIs [#edit-sli]
 
-Once you’ve created an SLI you can edit it either through the service levels list page, by clicking on the **...** menu and then `Edit`,
+Once you've created an SLI you can edit it either through the service levels list page, by clicking on the **...** menu and then `Edit`,
 
 <img
   title="Edit SLIs.png"
```

Comment:
@zuluecho9 while you're in the neighborhood, could you change the image title and alt text to something more helpful?